src/hotspot/share/jfr/leakprofiler/chains/jfrbitset.hpp

```diff
@@ -21,37 +21,13 @@
  * questions.
  *
  */
-#include "precompiled.hpp"
-#include "jfr/leakprofiler/chains/bitset.inline.hpp"
 
-BitSet::BitMapFragment::BitMapFragment(uintptr_t granule, BitMapFragment* next) :
-    _bits(_bitmap_granularity_size >> LogMinObjAlignmentInBytes, mtTracing, true /* clear */),
-    _next(next) {
-}
+#ifndef SHARE_JFR_LEAKPROFILER_JFRBITMAP_HPP
+#define SHARE_JFR_LEAKPROFILER_JFRBITMAP_HPP
```

Comment:
This is wrong - it should be BITSET not BITMAP

Reply:
> This is wrong - it should be BITSET not BITMAP

Oh, good find! I filed a follow-up issue:
https://bugs.openjdk.java.net/browse/JDK-8284725